actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -93,7 +95,9 @@ private void courseProgressMetricsReport(Request actorMessage) {
 
     String batchId = (String) actorMessage.getRequest().get(JsonKey.BATCH_ID);
     if (StringUtils.isBlank(batchId)) {
-      ProjectLogger.log("CourseMetricsActor-courseProgressMetrics-- batch is not valid .");
+      ProjectLogger.log(
+          "CourseMetricsActor:courseProgressMetricsReport  batch is not valid.",
```

Comment:
From:
"CourseMetricsActor:courseProgressMetricsReport  batch is not valid."

To:
"CourseMetricsActor:courseProgressMetricsReport: batchId is invalid (blank)."